explanation/intro-to/security.md

```diff
@@ -1,14 +1,79 @@
 (introduction-to-security)=
 # Introduction to security
 
-Security should always be considered when installing, deploying, and using any type of computer system. Although a fresh installation of Ubuntu is relatively safe for immediate use on the Internet, it is important to have a balanced understanding of your system's security posture based on how it will be used after deployment.
+Security should always be considered when installing, deploying, and using any
+Ubuntu system. Although a fresh installation of Ubuntu is relatively safe for
+immediate use, it is important to have a balanced understanding of your
+system's security posture based on how it will be used after deployment. It's
+important to take a layered approach so that your system's security is not
+dependent on a single 
 
-This chapter provides an overview of security-related topics as they pertain to Ubuntu Server Edition, and outlines simple measures you may use to protect your server and network from any number of potential security threats.
+## Server security guidance
 
-## About security at Ubuntu
+Since Ubuntu is so endlessly customizable, a full guide to security hardening
+is beyond the scope of this documentation.
+However, there are good practices and security-related packages that could be
+applied to almost any Ubuntu system. See our
+{ref}`security suggestions <security-suggestions>` page for an overview of the
+good habits and practices that can be adopted by anyone running an Ubuntu
+system to make it more secure. It's not necessary to apply every suggestion --
+and the list is not exhaustive by any means -- but each one used creates an
+extra layer of security.
+
+In a more advanced or complex setup, you may need to go further in your
+security outlook. There are specific packages available for your Server
+that will help with this, and we suggest some in the
+{ref}`advanced security <advanced-security>` section that you might want to
+consider for your use-case. Again, the list is not intended to be exhaustive,
+but rather a starting point.
+
+For a more thorough treatment of security in Ubuntu, we recommend checking
```

Comment:
"we also recommend" otherwise it sounds like read here is you ar not serious but there if you are.

Reply:
And sadly - as we know and as of today - there people find mostly marketing and not so much actionable